integration/integration_test.py

```diff
@@ -184,6 +184,15 @@ def _get_buffers(self):
 
 
 class IntegerType(PrimitiveType):
+    class Column(PrimitiveColumn):
```

Comment:
It isn't very idiomatic to declare nested classes, this should be `IntegerColumn` at the top level

Reply:
will do